visidata/column.py

```diff
@@ -473,6 +473,20 @@ def getMaxWidth(self, rows):
         w = min(w, self.sheet.windowWidth)
         return w
 
+    def getMaxDataWidth(self, rows):  #2255  need real max width for fixed width saver
```

Comment:
Let's move this function into the fixed_width.py for now (with `@Column.api`), since it's the only consumer so far.

Reply:
Ping @daviewales =)